### PR TITLE
QAE-826 - E2E test case for footer menu spacing

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Footer builder -> Footer menu -> Design -> Spacing

### How has this been tested?

npm run test:e2e:interactive tests/e2e/specs/customizer/footer-builder/elements/footer-menu/menu-spacing.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/utils/create-menu.js [hunk 1]
```diff
@@ -1,53 +1,24 @@
...
-	if ( await page.$( '.menu-delete' ) ) {
-		await page.click( '.menu-delete' );
-	}
+	await page.click( '.menu-delete' );
 	await page.focus( '#menu-name' );
-	await page.type( '#menu-name', 'Menu' );
-	await page.focus( '#locations-primary' );
-	await page.click( '#locations-primary' );
-	await page.focus( '#save_menu_footer' );
+	await page.type( 'Footer Menu' );
```
Comment (bsfsnehal) [inline tests/e2e/utils/create-menu.js:14]: @SimranChougule1 it showing error at local, please check.

Comment (SimranChougule1) [reply tests/e2e/utils/create-menu.js]: Please check now I resolved error

File: tests/e2e/utils/create-footer-menu.js [hunk 2]
```diff
@@ -2,22 +2,18 @@ import { createURL, createNewPost } from '@wordpress/e2e-test-utils';
 import { scrollToElement } from './scroll-to-element';
 import { setBrowserViewport } from '../../e2e/utils/set-browser-viewport';
 import { publishPost } from './publish-post';
-export const createNewMenu = async () => {
+export const createNewFooterMenu = async () => {
```
Comment (bsfsnehal) [inline tests/e2e/utils/create-footer-menu.js:5]: please create separate util for footer menu

Comment (SimranChougule1) [reply tests/e2e/utils/create-footer-menu.js:5]: created

**Review by GauravKhupse — APPROVED** [review]

**Review by bsfsnehal — APPROVED** [review]

**Pull request merged**